### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by naoei**
prereq: #5

### CODE REVIEW

File: GDEdit Web/GDE.Web/Data/GlobalVariables.cs [hunk 1]
```diff
@@ -5,70 +5,88 @@ namespace GDE.Web.Data
...
-        public enum LinkItems
-        {
-            Home,
-            Levels,
-            Community,
-            Help
-        }
+    }
+    
+    public enum LinkItems
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Data/GlobalVariables.cs:83]: Enum fields use PascalCase.

File: GDEdit Web/GDE.Web/Pages/GDApi.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using GDAPI.Application;
+using GDAPI.Objects.GeometryDash.General;
+using GDE.Web.Data;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace GDE.Web.Pages
+{
+    public class GDApiModel : PageModel
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Pages/GDApi.cshtml.cs:8]: `GDAPIModel`. Filename should be GDAPI.cshtml.

File: GDEdit Web/GDE.Web/Pages/GDApi.cshtml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿@page
+@model GDApiModel
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Pages/GDApi.cshtml:2]: `GDAPIModel`. Also file name should be GDAPI.cshtml.

File: GDEdit Web/GDE.Web/Startup.cs [hunk 4]
```diff
@@ -34,21 +38,32 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
...
 
-            app.UseEndpoints(endpoints =>
-            {
-                endpoints.MapBlazorHub();
-                endpoints.MapFallbackToPage("/_Host");
-            });
+            app.UseAuthentication();
+            app.UseAuthorization();
+
+            app.UseEndpoints(endpoints => { endpoints.MapRazorPages(); });
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Startup.cs:66]: The brackets are unnecessary; consider removing them. In general, avoid usage of delegates for multi-statement functions, rendering the brackets' usage useless altogether.

File: GDEdit Web/GDE.Web/Startup.cs [hunk 5]
```diff
@@ -15,14 +20,13 @@ public Startup(IConfiguration configuration)
         }
 
         public IConfiguration Configuration { get; }
-
+
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Startup.cs:23]: Unwanted whitespace.

File: GDEdit Web/GDE.Web/Properties/launchSettings.json [hunk 6]
```diff
@@ -1,10 +1,10 @@
-{
+﻿{
   "iisSettings": {
-    "windowsAuthentication": false,
-    "anonymousAuthentication": true,
+    "windowsAuthentication": false,
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Properties/launchSettings.json:3]: Unwanted whitespace at lines 3-4.

**Review by Rekkonnect — COMMENTED** [review]
In general, also replace `new []` with `new[]` (notice the lack of space between the two).

**Comment by Rekkonnect** [comment]
Also, FontAwesome addition should be split into another PR. This one lags to fucking oblivion.

File: GDEdit Web/GDE.Web/Data/GlobalVariables.cs (hunk 1 above)
Comment (naoei) [reply GDEdit Web/GDE.Web/Data/GlobalVariables.cs:83]: i guess i should use `.ToLowerInvariant`

**Pull request closed without merging**